src/main/java/redlightBack/Comment/Domain/Comment.java

```diff
@@ -0,0 +1,93 @@
+package redlightBack.Comment.Domain;
+
+import jakarta.persistence.*;
+import lombok.*;
+import redlightBack.common.BaseEntity;
+
+import java.time.LocalDateTime;
+
+
+@NoArgsConstructor
+@Getter
+@ToString
+@Entity
+public class Comment extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    //** post.id 간접 참조 **/
+    @Column(nullable = false)
+    private Long postId;
+
+    //** user.id 간접 참조 **/
+    @Column(nullable = false)
+    private String userId;
+
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "parent_id")
+    private Comment parent;
+
```

Comment:
`@JoinColumn(name = "parent_id")`을 생략해도 같아요.
String을 굳이 적어주지 않는 것이 '필드 이름 변경 시 String을 함께 변경하지 않는 실수' 발생 가능성을 없앨 수 있어서 더 좋지 않을까 싶습니다.
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    private Comment parent;
```